drivers/sensor/st/lsm6dsv16x/lsm6dsv16x_rtio_stream.c

```diff
@@ -0,0 +1,369 @@
+/* ST Microelectronics LSM6DSV16X 6-axis IMU sensor driver
+ *
+ * Copyright (c) 2023 Google LLC
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT st_lsm6dsv16x
+
+#include <zephyr/dt-bindings/sensor/lsm6dsv16x.h>
+#include <zephyr/drivers/sensor.h>
+#include "lsm6dsv16x.h"
+#include "lsm6dsv16x_decoder.h"
+#include <zephyr/rtio/work.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(LSM6DSV16X_RTIO);
+
+static uint64_t fifo_timestamp;
```

Comment:
my concern with having this static is that this would be shared with multiple `lsmdsv16x`, I would expect this to be within the driver data to prevent this

Reply:
Yes, of course. Don't know what is happening to me. Maybe too many things to do in parallel... Sorry